elasticsearch/full-text/src/main/java/nl/knaw/huc/api/MimetypeSubtypesResult.java

```diff
@@ -0,0 +1,42 @@
+package nl.knaw.huc.api;
+
+import nl.knaw.huc.MimetypeSubtypesConfiguration;
+import nl.knaw.huc.core.SupportedType;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MimetypeSubtypesResult {
+
+  public List<MimetypeSubtypeResult> types = new ArrayList<>();
+
+  /**
+   * Merge SupportedTypes and mimetypeSubtypes into a single List< MimetypeSubtype >
```

Comment:
Code clone. Maar ik zie ook even niet waar de gedeelde functionaliteit met de andere indexer dan heen zou moeten.
Krijg je op termijn misschien weer een 'library' (jar) met daarin 'hulpfunctionaliteit' o.i.d. vrees ik.
Voor nu maar als 'clone' laten?

Reply:
ja, zat ik zelf ook over na te denken... want meer van de indexer-functionaliteit is gedeeld: ik wilde dit nog eens met jou bespreken, hoe we hier het beste mee om kunnen gaan

Reply:
👍